components/ws-manager/pkg/manager/monitor.go

```diff
@@ -183,19 +183,20 @@ func (m *Monitor) onVolumesnapshotEvent(evt watch.Event) error {
 
 	// get the pod resource
 	var pod corev1.Pod
-	if err := m.manager.Clientset.Get(context.Background(), types.NamespacedName{Namespace: vs.Namespace, Name: podName}, &pod); err != nil {
-		log.WithError(err).Warnf("the pod %s/%s is missing", podName, vs.Namespace)
-		return nil
-	}
+	_ = m.manager.Clientset.Get(context.Background(), types.NamespacedName{Namespace: vs.Namespace, Name: podName}, &pod)
```

Comment:
hm. should we at least log any other error that might happen here except for not found error?

Reply:
fixed